### ISSUE TITLE
Remove Vega typings

### ISSUE THREAD

**Issue by domoritz** [vega/vega-lite#3413]
Use https://github.com/vega/vega-typings instead

**Comment by domoritz** [comment]
Blocked by vega/vega#1168

**Comment by kanitw** [comment]
Note that some of interfaces in VL may currently depend on Vega typings (with documentation).  


So we should also make sure to decouple this dependency so that we will preserve documentation in the JSON schema.

**Comment by domoritz** [comment]
Yes, the schema shouldn't change. We can move over piece by piece.

**Comment by domoritz** [comment]
Done

**Comment by kanitw** [comment]
Hmm, we still have `vega.schema.ts` -- better consolidate?
